src/vivarium_gates_mncnh/plugins/time.py

```diff
@@ -1,7 +1,8 @@
 import pandas as pd
 from vivarium.framework.engine import Builder
-from vivarium.framework.time import DateTimeClock
+from vivarium.framework.time import SimulationClock
```

Comment:
Not I was able to get this to run with subclassing simulation clock and not DateTimeClock